lib/ui/index.js

```diff
@@ -177,3 +183,17 @@ UI.prototype.writeLevelVisible = function(writeLevel) {
 
   return levels[writeLevel] >= levels[this.writeLevel];
 };
+
+/**
+  Whether or not the specified write level should be printed to the errorStream.
+
+  @method writeToErrorStream
+  @private
+  @param {String} writeLevel
+  @return {Boolean}
+*/
+UI.prototype.writeToErrorStream = function(writeLevel) {
+  writeLevel = writeLevel || 'INFO';
+
+  return writeLevel.match(/ERROR/i);
```

Comment:
@rwjblue if you're in favor of `===`, this method could be removed and the check can happen inline in the `write` method.